Gemfile

```diff
@@ -1,29 +1,31 @@
+# frozen_string_literal: true
+
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-gem 'rails', '~> 5.2.3'
-gem 'sqlite3'
+gem 'execjs'
```

Comment:
Is there a reason for all the reordering? I never noticed a relatec rubocop rule for this.

Reply:
Yes, there is a rubocop rule enforcing this.

https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Bundler/OrderedGems